.circleci/config.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - run:
           name: Acceptance Tests
           command: yarn acceptance src/test/acceptance/*.js
+      - run:
+          name: Cypress Tests
+          command: yarn test:smoke
```

Comment:
Maybe `test:cypress` will be clearer here - not sure `smoke` is the permanent functionality.

Reply:
Encoding library names in commands seems to leak domain-specific info, and ties commands to implementations. If what's being evaluated is (roughly) a smoke test (right now), then a developer encountering this command who is not familiar with cypress would have a heads up in debugging a problem, wheres otherwise it could be overlooked.

Reply:
in that case let's call it acceptance, and take over the nightmare naming.

Reply:
I had added `yarn cypress` and `yarn cypress:run` for convenience in running the Cypress test runner locally, in either GUI or headless mode.

But I'm def open to renaming the bash script and corresponding yarn script (which target a production build and would mostly be used only in CI)